src/axom/inlet/inlet_utils.cpp

```diff
@@ -12,5 +12,17 @@ void setWarningFlag(axom::sidre::Group* root) {
   }
 }
 
+std::string appendPrefix(const std::string& prefix, const std::string& name)
+{
+  return (prefix == "") ? name : prefix + "/" + name;
+}
+
+std::string removePrefix(const std::string& prefix, const std::string& name) {
+  SLIC_ASSERT_MSG(name.find(prefix) != std::string::npos, 

```

Comment:
https://github.com/LLNL/axom/blob/b156d86fae4c93e51eb247269c2ee70e08f9d246/src/axom/core/utilities/StringUtilities.hpp#L51

I should have linked you this before but please use this as its more accurate and already written.